content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -136,178 +117,88 @@ version, it's important to ensure that all existing CRs stored as v1 are migrate
 This migration can be achieved through _Storage Version Migration_ to migrate all CRs from v1 to v2.
 
 - Create a manifest for the CRD, named `test-crd.yaml`, as follows:
-  ```yaml
-  apiVersion: apiextensions.k8s.io/v1
-  kind: CustomResourceDefinition
-  metadata:
-    name: selfierequests.stable.example.com
-  spec:
-    group: stable.example.com
```

Comment:
These data implies that the snippet is only for demo's purpose, rather than something to be tried. So, I'd suggest leave them as inline YAML snippets.